README.md

```diff
@@ -8,31 +8,51 @@ On OS X, the best way to get it is to use homebrew:
 brew install remind101/formulae/assume-role
 ```
 
-If you have a working Go 1.6 environment:
+If you have a working Go 1.6/1.7 environment:
 
 ```bash
 $ go get -u github.com/remind101/assume-role
 ```
 
 ## Configuration
 
-The first step is to setup profiles for the different roles you'd like to assume in `~/.aws/config`. Follow the official AWS docs on how to do this at https://docs.aws.amazon.com/cli/latest/userguide/cli-roles.html.
+Setup a profile for each role you would like to assume in `~/.aws/config`.
 
-**Example**
+**For example, `~/.aws/config`:**
 
 ```ini
+[profile usermgt]
+# IAM user management AWS Account.
+region = us-east-1
+aws_access_key_id = AKIMYFAKEEXAMPLE
```

Comment:
Dunno if it's worth mentioning in this doc, but in general these would be put in `~/.aws/credentials`.